android/src/main/kotlin/com/bitmovin/player/flutter/FlutterPlayerView.kt

```diff
@@ -111,6 +119,28 @@ class FlutterPlayerView(
         sink = null
     }
 
+    private fun PlayerView.setOnPictureInPictureModeChanged(callback: (Boolean, Configuration) -> Unit) {
```

Comment:
```suggestion
    private fun PlayerView.setOnPictureInPictureModeChangedCallback(callback: (Boolean, Configuration) -> Unit) {
```
Where will this function be called from? I can't see any usage yet.

Reply:
It will be called from the init block where we check out the playerView arguments. Right after the fullscreen handler initialization.